shared/torngit/github.py

```diff
@@ -47,11 +50,61 @@
         }
     }
 }
-"""
+""",
+        REPOS_FROM_NODEIDS="""
+query GetReposFromNodeIds($node_ids: [ID!]!) {
+    nodes(ids: $node_ids) {
+        __typename 
+        ... on Repository {
+            # databaseId == service_id
+            databaseId
+            name
+            primaryLanguage {
+                name
+            }
+            isPrivate
+            defaultBranchRef {
+                name
+            }
+            owner {
+                # This ID is actually the node_id, not the ownerid
+                id
+                login
+            }
```

Comment:
do we need a separate query for getting owner information or could we include databaseId here?

Reply:
I would have included if it was possible

Reply:
RIP